documentation/docs/03-loading.md

```diff
@@ -98,6 +98,8 @@ So if the example above was `src/routes/blog/[slug].svelte` and the URL was `htt
 
 > When `fetch` runs on the server, the resulting response will be serialized and inlined into the rendered HTML. This allows the subsequent client-side `load` to access identical data immediately without an additional network request.
 
+> Cookies will only be passed through if the target host is the same as the SvelteKit application or a higher subdomain of it.
```

Comment:
To me "domain.com" is higher than "my.domain.com". Maybe we can use another word?

```suggestion
> Cookies will only be passed through if the target host is the same as the SvelteKit application or a more specific subdomain of it.
```